app/src/openxr/cpp/OpenXRLayers.cpp

```diff
@@ -242,4 +242,40 @@ OpenXRLayerEquirect::GetNextStructureInChain() const {
 }
 #endif
 
+// OpenXRLayerPassthrough;
+
+OpenXRLayerPassthroughPtr
+OpenXRLayerPassthrough::Create(const VRLayerPassthroughPtr& aLayer, XrPassthroughFB passthrough) {
+  auto result = std::make_shared<crow::OpenXRLayerPassthrough>();
```

Comment:
What about defining a constructor that receives 2 arguments, and then make this function something like this:

```return std::make_shared<crow::OpenXRLayerPassthrough>(aLayer, passthrough);```

Reply:
That would not be consistent with the rest of layer classes. It might make sense, but I think being consistent here is the whole point of these wrappers.

Reply:
Perhaps a comment suggesting a future refactoring would worth ?

Reply:
Ok